### PR TITLE
docs: replace screenshot with GIF

### LINKED ISSUE #787: [DOCS] Replace long screenshot with a GIF
### Image that needs to be replaced

Screenshot from 2021-12-09 08-36-24 

### Additional information

Profile -  http://linkfree.eddiehub.org/eddiejaoude

For Video to GIF you can use - https://www.onlineconverter.com/video-to-gif

### PR DESCRIPTION
**Pull request by LancelotHayward**
## Fixes Issue 787
Closes #787

## Changes proposed
Replace tall screenshot with short gif

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots
linkfree 787 slower

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/796"></a>

### CODE REVIEW

**Review by Pradumnasaraf — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 - LinkFree connects audiences to all of your content with just one link. It is an open-source alternative to [Linktree](https://linktr.ee/) implemented in JavaScript.
 - It was initially created on a YouTube [live stream](https://www.youtube.com/watch?v=Jorl_vcp-Ew).
 
-![Screenshot](https://user-images.githubusercontent.com/91655303/145024633-559c1a21-240b-4900-9f06-7917a6f0d241.png)
+![linkfree 787 slower](https://user-images.githubusercontent.com/82458069/145549448-b1feb5df-78aa-4286-a709-6985f0dade4c.gif)
```
Comment (Panquesito7) [inline README.md:6]:
```suggestion
![Eddie Jaoude's LinkFree profile GIF](https://user-images.githubusercontent.com/82458069/145549448-b1feb5df-78aa-4286-a709-6985f0dade4c.gif)
```

**Review by Panquesito7 — APPROVED** [review]

**Review by ssensalo — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -3,7 +3,7 @@
 - LinkFree connects audiences to all of your content with just one link. It is an open-source alternative to [Linktree](https://linktr.ee/) implemented in JavaScript.
 - It was initially created on a YouTube [live stream](https://www.youtube.com/watch?v=Jorl_vcp-Ew).
 
-![Screenshot](https://user-images.githubusercontent.com/91655303/145024633-559c1a21-240b-4900-9f06-7917a6f0d241.png)
+![Eddie Jaoude's LinkFree profile GIF](https://user-images.githubusercontent.com/82458069/145549448-b1feb5df-78aa-4286-a709-6985f0dade4c.gif)
```
Comment (mohan-murali) [inline README.md:6]: I think it will be better if we just say "Eddie Jaoude's LinkFree profile". No need to include GIF here.

**Review by krishguptadev — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (Pradumnasaraf) [reply README.md:6]: In my view, it's good to add a ***GIF*** label for better understandability, readability, and differentiating between GIF and Image.

**Review by Pradumnasaraf — APPROVED** [review]
LGTM

**Pull request merged**